### ISSUE TITLE
PreserveName inconsistency in debug script

### ISSUE THREAD

**Issue by robitar** [nikhilk/scriptsharp#45]
It seems that PreserveName doesn't get applied to fields in debug builds, it should be consistent otherwise you cant really use it as intended.

With the following class in C#:

```
public class MyClass {
    public string Property {
        get {
            return property;
        }
    }

    [PreserveName]
    private string property;
}
```

I get this output in debug builds, notice the private field is named "_property" with an underscore

```
o.MyClass = function o_MyClass() {
}
o.MyClass.prototype = {

    get_property: function o_MyClass$get_property() {
        return this._property;
    },

    _property: null
}
```

And I get this in release builds, now the field has the intended name of "property"

```
o.MyClass=function(){}
o.MyClass.prototype={get_property:function(){return this.property;},property:null}
```

**Comment by nikhilk** [comment]
Point noted.

Perhaps you can try [ScriptName(...)] in the meantime.

**Comment by robitar** [comment]
Thanks Nikhil, I'll give that a try, either that or just make the fields public for now.

**Comment by robitar** [comment]
Looks like ScriptName isn't being recognised in debug builds on non public members either

C# code

``` c#

using System.Runtime.CompilerServices;

namespace ScriptLibrary {

    public class Class1 {
        [ScriptName("prop")]
        private int prop;
    }
}
```

Debug script

``` js
ScriptLibrary.Class1 = function ScriptLibrary_Class1() {
}
ScriptLibrary.Class1.prototype = {
    _prop: 0
}
```

Release script

``` js
ScriptLibrary.Class1=function(){}
ScriptLibrary.Class1.prototype={prop:0}
```

**Comment by nikhilk** [comment]
Fix included in 0.7.2.0 release.

**Comment by robitar** [comment]
Thanks for the quick turnaround Nikhil
